### PR TITLE
Final edits changes to release v3.0.0 

### PR DESCRIPTION
**Pull request by PravinPK**

### CODE REVIEW

**Review by cacheung — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Integrate the Places mobile extension into your app by following the [getting st
 
 **Open the project**
 
-To open and run the project, open the `code/settings.gradle` file in Android Studio.
+To open and run the project, open the `code/build.gradle.kts` file in Android Studio.
```
Comment (cacheung) [inline README.md:19]: You can also keep to use code/setting.gradle.kts

File: code/gradle.properties [hunk 2]
```diff
@@ -24,5 +24,5 @@ mavenRepoDescription=Android Places Extension for Adobe Mobile Marketing
 mavenUploadDryRunFlag=false
 
 # production versions for production build
-mavenCoreVersion=3.0.0-beta.1-SNAPSHOT
+mavenCoreVersion=3.0.0
```
Comment (spoorthipujariadobe) [inline code/gradle.properties:27]: The `moduleVersion` is still 3.0.0-beta.1!

**Review by spoorthipujariadobe — CHANGES_REQUESTED** [review]

**Comment by spoorthipujariadobe** [comment]
Let's also remove `registerExtension` if it exists in the Github API docs

File: code/gradle.properties (hunk 2 above)
Comment (PravinPK) [reply code/gradle.properties:27]: good catch. thank you!

**Review by spoorthipujariadobe — APPROVED** [review]

**Pull request merged**